pkg/buildpipeline/test_data/stages_names_ok_with_sub_stages.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v0.1
+agent:
+  image: some-image
+stages:
+  - name: Stage A
+    steps:
+      - command: echo
+        args: ['last']
+  - name: Stage With Stages A
+    stages:
+      - name: A Working title 1
+        steps:
+          - command: echo
+            args:
+              - hello
+      - name: A Working title 2
+        steps:
+          - command: echo
+            args:
+              - hello
+  - name: Stage With Stages B
+    stages:
+      - name: A Working title 1
```

Comment:
I'm curious about what the generated tasks will look like for this example, see my comment in Jira.